jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -121,20 +124,27 @@ protected void doStart() throws Exception
         }
     }
 
+    /**
+     * Process the OpenID Connect metadata discovered by {@link #fetchOpenIdConnectMetadata()}.
```

Comment:
I was kind of hoping that you'd explain what processing can be done and what is done by default.
Currently it looks like you are just validating some parameters exist or are a certain value, but not much else.

So you need to at leaset explain that IAE can be  thrown, perhaps even add it to the signature